packages/utilities/src/aria.ts

```diff
@@ -4,9 +4,9 @@
  *
  * @param ariaAttributes - ARIA attributes to merge
  */
-export function mergeAriaAttributeValues(...ariaAttributes: (string | undefined)[]): string | undefined {
+export function mergeAriaAttributeValues(...ariaAttributes: (string | undefined | false)[]): string | undefined {
   const mergedAttribute = ariaAttributes
-    .filter((arg: string | undefined) => arg !== undefined && arg !== null)
+    .filter((arg: string | undefined | false) => arg !== undefined && arg !== null && arg !== false)
```

Comment:
can this be simplified to the following?

    .filter((arg: string | undefined | false) => arg)

I think the only change is that it ends up filtering out `''`, which seems fine.

Reply:
if i'm reading this right, it's filtering out undefined, null, and false. Not 0.

Reply:
strictly speaking, I'm not sure why we check for null when the types don't allow null. Likewise, 0 is technically an invalid input and even if someone does pass it in, we probably don't want to preserve "0". I will make the change JD suggested (what I had done here was just add on to the existing pattern but I think JD's suggestion is a net improvement)